pgsu/__init__.py

```diff
@@ -241,7 +240,7 @@ def _try_connect_psycopg(**kwargs):
     try:
         conn = connect(**kwargs)
         success = True
-        conn.close()
+        conn.close()  # pylint: disable=no-member
```

Comment:
Pylint started flagging these calls since `psycopg==3.2.4`. I have no idea why, I checked psycopg changelog and also run this code through several type checkers and none complained (also I think we would notice in aiida-core if there was such a breaking change).